Bottom: 5d39fe8bfb2d5e36e9b3592d7089c6fa93b975b2
Top:    5a00db801454baa6991a6b3be306f8b0a6e9455d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-23 20:55:35 +0000

Refresh of add-split-colouring-for-find-a

---

diff --git a/src/find.c b/src/find.c
index aa111f1..3622777 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3067,8 +3067,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int which_flag,
   if (type == NO_TYPE)
     {
       type = SearchObjectByLocation (
-        LOOKUP_MORE & ~(AndRats ? 0 /* RATLINE_TYPE */: /*0 
-                                                        */ RATLINE_TYPE),
+        LOOKUP_MORE & ~(AndRats ? 0 : RATLINE_TYPE),
         &ptr1, &ptr2, &ptr3, X, Y, Range);
       if (type == NO_TYPE)
         return;
